scripts/benchmark_stats.py

```diff
@@ -31,16 +32,19 @@ def _process_file_data(self):
             with open(file) as fp:
                 for row in DictReader(fp, delimiter=","):
                     percentile_99th = int(row["99%"])
+                    request_count = int(row.get("Request Count") or row.get("# requests"))
+
                     if "/questionnaire" in row['Name']:
                         if row["Type"] == "GET":
-                            self.get_requests.append(percentile_99th)
+                            self.get_requests.append(percentile_99th * request_count)
+                            self.total_get_requests += request_count
                         elif row["Type"] == "POST":
-                            self.post_requests.append(percentile_99th)
+                            self.total_post_requests += request_count
+                            self.post_requests.append(percentile_99th * request_count)
```

Comment:
Same as with the get requests. This should be renamed to `weighted_post_requests`